tests/run-tests.py

```diff
@@ -472,6 +472,13 @@ def run_tests(pyb, tests, args, result_dir, num_threads=1):
             for t in tests:
                 if t.startswith("basics/io_"):
                     skip_tests.add(t)
+        elif args.target == "renesas-ra":
+            skip_tests.add(
+                "basics/builtin_pow3_intbig.py"
```

Comment:
this test works and passes for me on a RA4M1 clicker board, so I think it should be included in the test run (ie this bit here removed)

Reply:
I also have confirmed that this test passes on a RA4M1 clicker board.  I included it in the test run.